packages/quantic/force-app/solutionExamples/main/lwc/resultAttachToCase/resultAttachToCase.js

```diff
@@ -0,0 +1,165 @@
+import {api, LightningElement, track} from 'lwc';
+import {
+  registerComponentForInit,
+  initializeWithHeadless,
+  getHeadlessBundle,
+} from 'c/quanticHeadlessLoader';
+// @ts-ignore
+import {buildResultToAttach, buildResultToDetach} from 'c/attachToCaseUtils';
+import attachResult from '@salesforce/label/c.AttachResult';
+import detachResult from '@salesforce/label/c.DetachResult';
+import resultIsAttached from '@salesforce/label/c.ResultIsAttached';
+// @ts-ignore
+import {attachToCase, detachFromCase} from 'c/attachToCaseServiceProd';
```

Comment:
Should this be
```suggestion
import {attachToCase, detachFromCase} from 'c/attachToCaseService';
```
?

Reply:
Indeed you are right I forgot to push my latest commit,  thank you!